BetterMD/elements/document.py

```
@@ -20,18 +55,21 @@ def copy(self) -> 'T1': ...
         self.data = data
-    
+
     def copy(self):
         return self.data
 
 T5 = t.TypeVar("T5", bound=CopyProtocol)
+HASHABLE_ATTRS = str | bool | int | float | HashableList['HASHABLE_ATTRS'] | HashableDict[str, 'HASHABLE_ATTRS']
 
 class Fetcher(t.Generic[T1, T2, T5]):
-    def __init__(self, data: 'GetProtocol[T1, T2]', default:'T5'=Copy(None)):
+    def __init__(self, data: 't.Union[GetProtocol[T1, T2], dict[T1, T2]]', default:'T5'=Copy(None)):
```

Ruff (0.8.2)

Do not perform function call `Copy` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)